app/views/shared/fields/_copyright.html.erb

```diff
@@ -24,7 +26,16 @@
 			<p><strong><%= copyright['status'].gsub(/--/, ' – ') %></strong></p>
 		<% end %>
 		<p><%= copyright['purposeNote'] %></p>
-		<p><%= copyright['note'] %></p>
+    
+    <% if unit_code == "rdcp" %>
```

Comment:
I think a test demonstrating this change _might_ be helpful, primarily in the future when we need to migrate to a new application. Basically just something demonstrating that we're doing something unique with respect to RDCP copyright. Do others have an opinion on this?

Reply:
Thanks for the suggestion. It's not hard to write the test for it. But I feel this is not like VRR updates, it's just a trivial view changes in the .erb file. It will be too much tests in our codebase if add such test suit case. What do you think?

Reply:
Yeah, that's why I said might. It feels on the fence. The reason I lean towards a test is this change is, in a sense, business logic. Albeit fairly trivial business logic :) 

My real concerns are:
- losing track of stuff like this in the future. But it is so small, at this point, that I think your pushback is fair. 
- there being more "If RDCP then do this, else do that" stuff coming in the future. My current branch waiting on Solr reindexing, for example, does this for sorting. 

So like I said, I don't have a strong opinion we need a test or two for this, but those are the reasons I mention it. @lsitu and @VivianChu , what are your thoughts?

Reply:
I don't have a strong opinion for a test either.  However, I think it would be nice to have a test since the change is business logic.